### DISCUSSION TITLE
[Kontoverwaltung] Übersicht Jahrgänge Admin

### DISCUSSION THREAD

**Discussion by Schmarvinius** [SE-TINF22B2/G2-Schoolify#126] [User Stories]
## User Story:
Als Admin möchte ich eine Übersicht über alle Jahrgänge haben.

## Beschreibung
Es soll eine Liste aller Jahrgänge unterteilt in die einzelnen Klassen geben. Die Klassen sollen Informationen über die Schüler darin geben.
Dazu sollte es eine Schüler Suche geben.

## Akzeptanz-Kriterien
- [ ] Übersicht der Jahrgänge existiert
- [ ] Die Klassen befinden sich in den zugehörigen Jahrgängen
- [ ] Schüler sind in den Klassen einsehbar

### Issues
- #139

**Comment by laralngn** [comment]
Nutzen fehlt
